src/form/birth/index.ts

```diff
@@ -309,63 +346,65 @@ export const birthForm: ISerializedForm = {
               mothersDetailsExistConditionals
             ),
             getReasonNotExisting(certificateHandlebars.motherReasonNotApplying), // Strongly recommend is required if you want to register abandoned / orphaned children!
-            getFirstNameField(
-              'motherNameInEnglish',
-              motherFirstNameConditionals,
-              certificateHandlebars.motherFirstName
-            ), // Required field.
-            getFamilyNameField(
-              'motherNameInEnglish',
-              motherFamilyNameConditionals,
-              certificateHandlebars.motherFamilyName
+            getNationality(
+              certificateHandlebars.motherNationality,
+              detailsExist
             ), // Required field.
+            getNationalID(
+              'iD',
```

Comment:
'ID' or 'id' maybe?  let me know if intentionally `iD`

Reply:
unfortunately yes, there are several places where we are maintaining this case 😞